### DISCUSSION TITLE
Question related clipboard idk if it is bug

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [florisboard/florisboard#1957] [General]
Copy some queries.
Set one to pin
Remove all queries
When u select & click on paste last copied query will get pasted(while it is not in clipboard)

**Comment by patrickgold** [comment]
Primarily clip IS NOT the same as the clipboard history, thus this behavior occurs. The history is a FlorisBoard internal database, while the primary clip is either managed and stored by the system or FlorisBoard.

**Reply by HikaruIchijyo** [reply]
![Screenshot_2022-07-16-09-33-48-910_dev.patrickgold.florisboard.beta.jpg](https://user-images.githubusercontent.com/38470944/179339474-29efa948-3c19-4a7d-a3b4-80319dafd477.jpg)

Oh then what is the workaround?

**Reply by patrickgold** [reply]
Tbh there is no direct workaround, the sync is currently only really implemented in the direction primary clip > history and not the other way around. I don't know though if a feature request for your use case already exists.

**Reply by HikaruIchijyo** [reply]
> Tbh there is no direct workaround, the sync is currently only really implemented in the direction primary clip > history and not the other way around. I don't know though if a feature request for your use case already exists.

Oh
